zookeeper-metrics-providers/pom.xml

```diff
@@ -37,4 +37,41 @@
     <module>zookeeper-prometheus-metrics</module>
   </modules>
 
+  <build>
+    <pluginManagement>
+      <plugins>
+        <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-checkstyle-plugin</artifactId>
+          <version>3.0.0</version>
```

Comment:
3.1.0 please

Reply:
Also updated maven-checkstyle-plugin version in pom.xml and zookeeper-jute/pom.xml